internal/bloblang/query/functions.go

```diff
@@ -653,6 +662,14 @@ func randomIntFunction(args *ParsedParams) (Function, error) {
 	if err != nil {
 		return nil, err
 	}
+	min, err := args.FieldInt64("min")
+	if err != nil {
+		return nil, err
+	}
+	max, err := args.FieldInt64("max")
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Might be worth checking somewhere here that `max - min >= 0`. Otherwise, `Int63n` will panic for such cases.

Reply:
Added validations to make sure min is positive and max if equal or bigger than min